test_runner/src/test/kotlin/ftl/cli/firebase/test/android/configuration/AndroidLocalesListCommandTest.kt

```diff
@@ -0,0 +1,16 @@
+package ftl.cli.firebase.test.android.configuration
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import picocli.CommandLine
+
+class AndroidLocalesListCommandTest {
+
+    @Test
+    fun androidLocalesListCommandShouldParseConfig() {
+        val cmd = AndroidLocalesListCommand()
+        CommandLine(cmd).parseArgs("--config=a")
+
+        assertThat(cmd.configPath).isEqualTo("a")
```

Comment:
If it makes a sense, maybe we can test if the command runs what we expected? What do you think?

Reply:
I have added tests which you suggested

Reply:
👍